samples/grid-pro/demo/chart-from-column-grid/demo.html

```diff
@@ -0,0 +1,12 @@
+<script src="https://code.highcharts.com/dashboards/datagrid.js"></script>
+<script src="https://code.highcharts.com/highcharts.js"></script>
+
+<div class="content">
+    <div id="container">
+        <div id="grid"></div>
+        <div id="chart"></div>
+    </div>
+    <p class="highcharts-description">
+        This demo shows how to plot a column from a Grid in a chart
```

Comment:
I had to look at the code to figure out that you have to click on the cell to see the chart. Please describe it here.